### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by valllllll2000**
#### Before the PR (what is the issue or what needed to be done)
NA
#### The solution
PR template updated with new more explanatory bullet points.

In the future, would be great if 2, 3, 4 and 5 were checked automatically
#### Screenshots (if appropriate)
NA
## Checklist
* [ ] Connect the issue
* [ ] Test plan
* [ ] Copyright header
* [ ] Code formatting
* [ ] Documentation

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-## Checklist
-* [ ] Connect the issue
-* [ ] Test plan
-* [ ] Copyright header
-* [ ] Code formatting
-* [ ] Documentation
+#### Reviewer Checklist
+* [ ] Added an explanation about the work done
+* [ ] Connected the PR and the issue on Zenhub
+* [ ] Added a test plan to the issue
```
Comment (muloem) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe we should modify this to also point out that automated tests should be added  as well if necessary?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 #### Screenshots (if appropriate)
 
-## Checklist
-* [ ] Connect the issue
-* [ ] Test plan
-* [ ] Copyright header
-* [ ] Code formatting
-* [ ] Documentation
+#### Reviewer Checklist
+* [ ] Added an explanation about the work done
```
Comment (muloem) [inline .github/PULL_REQUEST_TEMPLATE.md]: This is already part of the template no? i.e. the section _The solution_

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (valllllll2000) [reply .github/PULL_REQUEST_TEMPLATE.md]: yes good idea!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (valllllll2000) [reply .github/PULL_REQUEST_TEMPLATE.md]: it is but this is a reminder for the reviewer and also for the person who creates it to make sure they filled out the information

**Pull request merged**